### PR TITLE
GEODE-5971: Refactor various commands to extend GfshCommand and extend ResultModel

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
- DebugCommand
- ExportStackTraceCommand
- GfshHelpCommand
- GfshHintCommand

Authored-by: Jens Deppe <jdeppe@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jdeppe-pivotal** [comment]
@Petahhh Please review

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DebugCommand.java [hunk 1]
```diff
@@ -19,17 +19,16 @@
...
-import org.apache.geode.management.cli.Result;
+import org.apache.geode.management.cli.GfshCommand;
 import org.apache.geode.management.internal.cli.i18n.CliStrings;
-import org.apache.geode.management.internal.cli.result.ErrorResultData;
-import org.apache.geode.management.internal.cli.result.ResultBuilder;
+import org.apache.geode.management.internal.cli.result.model.ResultModel;
 import org.apache.geode.management.internal.cli.shell.Gfsh;
 
-public class DebugCommand extends InternalGfshCommand {
+public class DebugCommand extends GfshCommand {
```
Comment (jinmeiliao) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DebugCommand.java]: We should extend OfflineGfshCommand for all offline commands. It has getGfsh() methods to get the gfsh isntance.

Comment (jdeppe-pivotal) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DebugCommand.java]: Right. Thanks!

File: geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ClientClusterManagementServiceWithSslDunitTest.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/*
```
Comment (jinmeiliao) [inline geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ClientClusterManagementServiceWithSslDunitTest.java]: you don't mean to add this, right?

Comment (jdeppe-pivotal) [reply geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ClientClusterManagementServiceWithSslDunitTest.java]: Right!

**Pull request merged**